src/web/src/router/index.js

```diff
@@ -81,7 +186,7 @@ router.beforeEach(async (to, from, next) => {
   var isAuthenticated = store.getters.isAuthenticated;
 
   if (requiresAuth && !isAuthenticated) {
-    console.log("You aren't authenticatd, redirecting to sign-in")
+    console.log("You aren't authenticatd, redirecting to sign-in");
```

Comment:
Typo in "authenticatd" and many changes above set `requiresAuth` to false, so that's a less-restrictive permission change. Intended as such?